npm/webpack-preprocessor/lib/cross-origin-callback-compile.ts

```diff
@@ -0,0 +1,76 @@
+import _ from 'lodash'
+import Debug from 'debug'
+import * as path from 'path'
+import webpack from 'webpack'
+import { CrossOriginCallbackStoreFile } from './cross-origin-callback-store'
+
+const VirtualModulesPlugin = require('webpack-virtual-modules')
+
+const debug = Debug('cypress:webpack')
+
```

Comment:
I feel documenting the methods in this file would go a long way to make it a bit clearer what is happening. I had to give the tests in the driver a run to get a decent handle on that, but I also feel like that is expected?

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/23283/commits/a3826c68871558f1c1808fc8ad17bc63d532ad3d